automation/build-bin.ts

```diff
@@ -348,7 +348,9 @@ async function getSubprocessStdout(
 		child.stderr.on('data', (data: Buffer) => {
 			try {
 				const stderr = data.toString();
-				reject(new Error(`"${execPath}": non-empty stderr "${stderr}"`));
+				if (!stderr.trim().startsWith('[debug]')) {
+					reject(new Error(`"${execPath}": non-empty stderr "${stderr}"`));
+				}
```

Comment:
Hmm I'm not sure if the 'data' event is necessarily triggered one text line at a time... If it was then this would be fine, but if it could create a 'data' event with, say, 2 lines as follows, then a real error would be ignored:
```
[debug] First line can be ignore, it's a debug message
Error: unable to create standalone zip package because this PR broke it
```
If `data.toString()` consisted of the 2 lines above, the error would be ignored.